net/samba4/Portfile

```diff
@@ -1,50 +1,61 @@
 PortSystem 1.0
 
 name			samba4
-version			4.0.0tp5
-revision		5
+version			4.13.0
 categories		net
 platforms		darwin
 maintainers		nomaintainer
 license			GPL-3
-description		SMB/CIFS server and client, ALPHA release
+description		SMB/CIFS server and client
 long_description	Samba is an software suite that provides seamless file \
-			and print services to SMB/CIFS clients. \
-			This is an ALPHA release!
+			and print services to SMB/CIFS clients.
 
 homepage		http://www.samba.org/
-master_sites		http://download.samba.org/samba/ftp/samba4/ \
-			ftp://de.samba.org/samba.org/samba4/
+master_sites		https://download.samba.org/pub/samba/stable
 distname		samba-${version}
-checksums           md5     dbc2e04455d67b01d2020808065b96df \
-                    sha1    8897cc7a824a594b3543245cdbbceaea32e9e08d \
-                    rmd160  1870f67aa851751e381a3b03dadfa1d0afa28a12
+checksums		rmd160  0e0efa2be30af3df3cce92b00496b296cc4c2a0d \
+			sha256  f11d52aee8db0aa50ac614143f2ccefead29e8f8ff585358792221557cade7f6 \
+			size	18406380
+
+depends_lib		port:cctools \
+			port:gettext \
+			port:gnutls \
+			port:gpgme \
+			port:jansson \
+			port:libarchive \
+			port:libiconv \
+			port:openldap \
+			port:p5.28-parse-yapp \
+			port:popt \
+			port:python38 \
+			port:readline \
+			port:zlib
+
+patch.pre_args  -Np1
```

Comment:
Why use the `-N` flag, which means "Ignore patches that seem to be reversed or already applied." Why would we want to ignore that?

Reply:
I usually use this, because a patch might be merged into the code in the next version, and break the build process. I personally don't see the point in stopping a build because a patch has already been merged into mainstream, but feel free to remove that if you see fit.

Reply:
We should remove `-N`. We want to be alerted to any patch failures so that we can correct the problem by updating or removing the patch.